oat-sa/extension-tao-itemqti-pci
# Fix/TAO-10102/liquid interaction preview

**Related to task:** https://oat-sa.atlassian.net/browse/TAO-10102
**Description:** Item preview on result page is not showing submitted answer
**Requires:** 
 - [x] To be released https://github.com/oat-sa/extension-tao-itemqti/pull/1420

**Steps to reproduce:** 
- Create Liquids PCI
- Create and add item to test
- Create delivery, pass the test.
- Check test results on "Results" page click on review button.
